cmd/ebrelayer/txs/signature.go

```diff
@@ -8,19 +8,13 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/joho/godotenv"
 	solsha3 "github.com/miguelmota/go-solidity-sha3"
 
 	"github.com/Sifchain/sifnode/cmd/ebrelayer/types"
 )
 
 // LoadPrivateKey loads the validator's private key from environment variables
 func LoadPrivateKey() (key *ecdsa.PrivateKey, err error) {
-	// Load config file containing environment variables
-	if err := godotenv.Load(); err != nil {
-		log.Fatal("Error loading .env file", err)
-	}
```

Comment:
So now there is no more config file like we discussed and there is only environment variables that we read in

Reply:
Yes, I'm not a fan of having a `.env` file floating around in a production environment. I feel it's better to set the environment variables when spinning up the environment.

Reply:
Yeah I would gitignore all `.env` file and then they can only be used for local development.